.gitignore

```diff
@@ -1,10 +1,9 @@
-vendor
-composer.lock
-composer.phar
-phpunit.xml
-phpmd.xml
-phpdox.xml
-clover.xml
-example/test-data
-phing-latest.phar
-build/
+/build/
```

Comment:
We probably only want to ignore files in the root directory.

Appended `/` to make it more obvious which is expected to be a directory and which a file.

Sorted by

- kind (directory or file)
- name